lerobot-web-fe/src/services/robot.service.ts

```diff
@@ -1,9 +1,13 @@
-import { API_BASE } from './status.service';
+import { useStatusStore } from '../stores/status.store';
 import type { JointStatesResponse } from '../models/robot.model';
 
 export async function getJointPositions(id: number): Promise<JointStatesResponse> {
+  const { apiUrl } = useStatusStore.getState();
```

Comment:
Instead of destruct, please use the separated selector-based solution here as well. Thank you!

```suggestion
  const apiUrl = useStatusStore((s) => s.apiUrl);
```

Reply:
It will cause an invalid hook call. The useStatusStore(s => s.apiUrl) is for reactive, efficient updates. The  useStatusStore.getState().apiUrl usage is safe, static, and meets React's hook rules.